python/cudf/cudf/core/frame.py

```diff
@@ -548,6 +547,17 @@ def to_cupy(
         -------
         cupy.ndarray
         """
+        if self.ndim == 1:
```

Comment:
Could this be defined in `SingleColumnFrame`? It's the only class where this holds true

Reply:
Yup, moved it